imports/ui/components/account/AccountAlerts.js

```diff
@@ -33,9 +33,18 @@ export const errorMsgs = {
   ERR_EMAIL_UNRECOGNIZED: "is not recognized. Have you created an account yet?",
   ERR_EMAIL_UNVERIFIED: "Your e-mail has not been verified. Please check",
   ERR_ACCOUNT_LOCK: "Your account has been locked. Please check",
-  ERR_EMAIL_ENTERED_INVALID: "Invalid NUS e-mail. Remember to end your e-mail address with '@u.nus.edu'",
+  ERR_EMAIL_ENTERED_INVALID_DOMAIN: "Invalid NUS e-mail. Remember to end your e-mail address with '@u.nus.edu'",
+  ERR_EMAIL_ENTERED_INVALID_FORMAT:"Invalid email format. Please check if there are invalid puncutations in your email",
   ERR_SETUP_INCOMPLETE: "Please enter all three fields before continuing"
 }
+export const loadingMsgs = {
```

Comment:
Good to separate the loading messages from the success/fail messages for better readability and maintainability!